content/en/docs/reference/config-api/kube-proxy-config.v1alpha1.md

```diff
@@ -0,0 +1,537 @@
+---
+title: kube-proxy Configuration (v1alpha1)
+content_type: tool-reference
+package: kubeproxy.config.k8s.io/v1alpha1
+auto_generated: true
+---
+
+
+## Resource Types 
+
+
+  
+- [KubeProxyConfiguration](#kubeproxy-config-k8s-io-v1alpha1-KubeProxyConfiguration)
+  
+    
+
+
+## `KubeProxyConfiguration`     {#kubeproxy-config-k8s-io-v1alpha1-KubeProxyConfiguration}
+    
+
+
+
+
+KubeProxyConfiguration contains everything necessary to configure the
+Kubernetes proxy server.
+
+<table class="table">
```

Comment:
Howdy, @tengqm! 👋🏼 

For this block of documentation, would you be able to change this to Markdown format, or were there any items that weren't formatting correctly and need to be in HTML?

Reply:
Markdown tables are weak on formatting. It is actually error prone. Generating markdown tables means we have to carefully filter out all contents we extract from the comment in the GoLang source code. Consider the complexity of parsing and removing line breaks, I'd suggest we stick to HTML tables.